backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -244,10 +247,10 @@ async def upload_facilities_excel_sheet(
 @router.post('/facilityWithSupervisors',
              summary='Upload and process facility with supervisors Excel file',
              response_description="Returns processed Excel file with validation results")
-async def upload_boundaries_excel_sheet(
+async def upload_facility_with_supervisors_excel_sheet(
         facility_with_supervisors: UploadFile = File(
             description="Excel file containing facility with supervisors data"),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)